apps/tangle-dapp/types/index.ts

```diff
@@ -141,6 +141,11 @@ export enum ServiceType {
   DKG_TSS_CGGMP = 'DKG/TSS (CGGMP)',
 }
 
+export enum ProfileType {
+  INDEPENDENT = 'Independent',
+  SHARED = 'Shared',
+}
+
```

Comment:
I've already implemented this on my PR, but should be fine. Once the merge occurs, should be resolved.